app/scripts/components/exploration/hooks/use-stac-metadata-datasets.ts

```diff
@@ -94,9 +94,13 @@ async function fetchStacDatasetById(
     `${stacApiEndpointToUse}/collections/${stacCol}`
   );
 
+  const timeDensity =
+    data['dashboard:time_density'] || time_density || TimeDensity.DAY;
   const commonTimeseriesParams = {
     isPeriodic: !!data['dashboard:is_periodic'],
-    timeDensity: data['dashboard:time_density'] || TimeDensity.DAY
+    timeDensity: timeDensity,
+    timeInterval:
+      data['dashboard:time_interval'] || `P1${timeDensity[0].toUpperCase()}`
```

Comment:
Is the plan to deprecate `dashboard:time_density`?

Reply:
> Is the plan to deprecate `dashboard:time_density`?

yes, but incrementally.

Hopefully this takes care of that https://github.com/NASA-IMPACT/veda-ui/pull/1560/commits/937d7cd7fb84bfa3d6d13bf351aa4e0098f3c41d